src/fetch.js

```diff
@@ -0,0 +1,13 @@
+import showWeather from './display';
+import api from './utility';
+
+function fetchData(country) {
+  const url = `http://api.openweathermap.org/data/2.5/weather?q=${country}&appid=${api}`;
+  fetch(url, { mode: 'cors' })
```

Comment:
[Required]

- [ ] It uses either promises or async/await ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/promises_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app)).
As it will help you in handling error if fetching request receive any kind of error.